### DISCUSSION TITLE
Using Latex within Quarto

### DISCUSSION THREAD

**Discussion by nipnipj** [quarto-dev/quarto-cli#2637] [Q&A]
How can I use Latex code within Quarto? I mean in a similar way to Rmarkdown.

**Comment by melissavanbussel** [comment]
You can insert math into a Quarto document using dollar signs, just like in R Markdown!

```
$$
x^2
$$
```

To add in-line math, just use single dollar signs instead of double. For example, `$x^2$`.

**Comment by nipnipj** [comment]
What about writing tables. like in 
```
\begin{table}
...
\end{table}
```

**Reply by melissavanbussel** [reply]
Here's the page that describes how include raw LaTeX in your Quarto documents: [https://quarto.org/docs/output-formats/pdf-basics.html#raw-latex](https://quarto.org/docs/output-formats/pdf-basics.html#raw-latex)

You can include raw LaTeX in your PDF documents, but raw LaTeX will be ignored when rendered to HTML or Word. For including tables, you probably want to use Markdown tables. 

If you want to write your tables using LaTeX, you could always convert the LaTeX table to Markdown if you're trying to render to something other than PDF!

**Comment by nipnipj** [comment]
It is possible to use Latex within quarto to create .pdf files. Can we do the same to create .html files?  I'm particularly interested in the creation of nice tables using latex packages like `threeparttable`.

**Reply by dragonstyle** [reply]
Aside from math, no. LaTeX directives are ignored for HTML and other non-PDF output types.
